packages/react-components/src/types/ChatMessage.ts

```diff
@@ -61,6 +61,7 @@ export interface ChatMessage extends MessageCommon {
   senderId?: string;
   senderDisplayName?: string;
   status?: MessageStatus;
+  failureReason?: string;
```

Comment:
Curious about where does this failureReason coming from? From our state.error or somewhere else

Reply:
It's the prop to provide the context to show in the UI. It will be provided by wherever raises the error, which in the edit case would be when the onUpdateMessage callback throws an error.